### PR TITLE
Accelerometer integration for iOS

### PR DESCRIPTION
**Pull request by AgentSmax**

### CODE REVIEW

**Comment by AgentSmax** [comment]
This branch has @themuflon's changes in them as well since I branched out of his branch. So I suppose this can be merged if his changes are ok and final.

**Comment by SmilyOrg** [comment]
There are still some questions I'd like to resolve in #497 first and some of them also affect this PR, so lets get that settled before landing either :)

File: application/android/src/co/theengine/loomplayer/LoomSensors.java [hunk 1]
```diff
@@ -192,14 +187,15 @@ public void onSensorChanged(SensorEvent event)
                     ///if the sensor data is unreliable return
                     if(event.accuracy == SensorManager.SENSOR_STATUS_UNRELIABLE)
                     {
-                        return;
+                     //   return;
```
Comment (bengarney) [inline application/android/src/co/theengine/loomplayer/LoomSensors.java]: Dead code?

**Comment by bengarney** [comment]
One nit.

**Comment by AgentSmax** [comment]
I forgot, there are also nits that need to be resolved in #497 first because this was branched off of that branch. That was done because it already had some fixes that were needed here.

In hindsight I probably shouldn't have closed the PR :P

**Comment by AgentSmax** [comment]
I just now realized that the nit was not in my scope of changes and was there because of this branch being branched out from #497.

Rebased to master.

**Comment by SmilyOrg** [comment]
Nice!

**Pull request merged**